CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-# 4.20.1 (2021-07-19)
+# 4.20.1 (2021-07-20)
 
 - Fixed the selection box component to properly style selection boxes that are disabled after initialization. [#280](https://github.com/blackbaud/skyux-forms/pull/280)
+- Fixed the input box component to prevent overlap between the label and the textarea element. [#279](https://github.com/blackbaud/skyux-forms/pull/279)
```

Comment:
```suggestion
- Fixed the input box component to prevent overlap between the label and the `textarea` element. [#279](https://github.com/blackbaud/skyux-forms/pull/279)
```